api/src/api/graphql/translate/dictionary.clj

```diff
@@ -38,3 +39,70 @@
                         :label  ""
                         :roles  []
                         :frames []}))}))
+
+(defn pos->schema [pos]
+  (case pos
+    :n "NN"
+    :adv "RB"
+    :adj "JJ"
+    :v   "VB"
+    :v2  "VB"
+    :sconj "IN"
+    (name pos)))
+
+(defn multilang-dict-item->schema [{:keys [id key pos gender language senses inflections tenses] :as dict-item}]
+  (log/debugf "MultilangDictItem: %s" dict-item)
+  {:id            id
+   :key           key
+   :pos           (pos->schema pos)
+   :language      (name language)
```

Comment:
Review all uses of language and lets stick to GF use of them, that is a string of three letters "Eng", "Ger", "Lav", "Rus", "Est"